Makefile

```diff
@@ -55,22 +55,17 @@ install:	all
 # Formats all .h and .cpp file under the current directory
 # It assumes clang is available on the PATH and will fail otherwise
 formatAll:
-	clang-format -i --style=file $(shell find . -not \( -path ./vendor -prune \) -type f \( -name "*.h" -o -name "*.cpp" -o -name "*.c" \))
+	@clang-format -i --style=file $(shell ./scripts/which_clang_files all)
```

Comment:
Whats the issue with echoing from clang-format? I kinda like it. Is it just too long in this case?

Reply:
You added an `@` prefix which I understood to surpress echoing the command

Reply:
ohhh right. Yeah it was just a bit long/inconsistent with the `make format` I don't have a strong preference either way

Reply:
This is fine just wanted to know why. all good